frontend/fingertips-frontend/components/organisms/ViewsWrapper/hasSufficientHealthDataCheck.ts

```diff
@@ -1,34 +1,43 @@
 import { IndicatorWithHealthDataForArea } from '@/generated-sources/ft-api-client';
 
-const hasAreaHealthData = (
+const hasSufficientAreaHealthData = (
```

Comment:
The same variable name is used on line 3 and line 16 - could we choose an alternative to make things clearer.

Reply:
Yeah updating naming has helped. Good spot 👍